crates/client/src/config.rs

```diff
@@ -110,6 +110,10 @@ pub struct Config {
     /// List of creds availabe in keyring
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub keys: Option<IndexSet<String>>,
```

Comment:
While this is preexisting, this should be `IndexSet<String>` with `skip_serializing_if` set to `IndexSet::is_empty`.

Having it wrapped in an option is a bit unwieldy.

Reply:
Oh thanks for the rec I think this will actually make a lot of the keyring code a lot simpler.